### PR TITLE
Integrate CiderExecutionKernel

### PR DESCRIPTION
**Pull request by jikunshang**

### CODE REVIEW

**Comment by yma11** [comment]
looks good to me.

File: velox/exec/tests/PlanBuilder.cpp [hunk 1]
```diff
@@ -16,11 +16,12 @@
 #include "velox/exec/tests/PlanBuilder.h"
 #include <velox/exec/Aggregate.h>
 #include <velox/exec/HashPartitionFunction.h>
-#include <velox/exec/HybridExecOperator.h>
 #include "velox/connectors/hive/HiveConnector.h"
 #include "velox/parse/Expressions.h"
 #include "velox/parse/ExpressionsParser.h"
 
+#include <velox/exec/HybridExecOperator.h>
```
Comment (winningsix) [inline velox/exec/tests/PlanBuilder.cpp:23]: Add a note here and fixme to remind.

**Review by winningsix — APPROVED** [review]
LGTM. Some minor comments.

File: velox/exec/HybridExecOperator.h [hunk 2]
```diff
@@ -18,6 +18,9 @@
 #include "velox/exec/Operator.h"
 #include "velox/core/HybridPlanNode.h"
 
+// include order matters
```
[the remark this answers is not in the document]
Comment (jikunshang) [reply velox/exec/HybridExecOperator.h]: Done.

**Pull request merged**